Tools/scripts/build_autotest.sh

```diff
@@ -58,9 +58,8 @@ echo "Updating APM"
 pushd APM
 git checkout -f master
 git fetch origin
-git submodule update --recursive --force
 git reset --hard origin/master
-git pull
+git submodule update --recursive --force
```

Comment:
please add --init
or call Tools/gittools/submodule-sync.sh

Reply:
Done and tested